Tests/configure_and_test_integration_instances.py

```diff
@@ -123,7 +123,8 @@ class Build:
     test_pack_target = '{}/project/Tests'.format(os.getenv('HOME'))
     key_file_path = 'Use in case of running with non local server'
     run_environment = Running.CIRCLECI_RUN
-    env_results_path = './artifacts/env_results.json'
+    env_results_path = os.getenv('ENV_RESULTS_PATH', os.path.join(os.getenv('ARTIFACTS_FOLDER', './artifacts'),
```

Comment:
backups on backups on backups... 😃

Reply:
its for the private build, they use ARTIFACTS_FOLDER/env_results.json